### ISSUE TITLE
finally accuracy

### ISSUE THREAD

**Issue by ButIprferit** [yuhuixu1993/PC-DARTS#30]
hi，I want to know train how many epoch can get the 2.5% error in cifar10 ？
thank you vary much！

**Comment by chenxin061** [comment]
We followed the training setting of DARTS and train the discovered architecture for 600 epochs.
